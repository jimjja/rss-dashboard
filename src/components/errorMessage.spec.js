import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './errorMessage';

describe('Error Message component', () => {
  it('should match snapshot', () => {
    const props = {
      label: 'Error message',
      text: 'Failed to fetch',
    };
    const tree = renderer.create(<ErrorMessage {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should not render label', () => {
    const props = {
      text: 'Failed to fetch',
    };
    const tree = renderer.create(<ErrorMessage {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
