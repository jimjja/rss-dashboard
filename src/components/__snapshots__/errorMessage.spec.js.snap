// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Message component should match snapshot 1`] = `
<p
  style={
    Object {
      "color": "red",
    }
  }
>
  <strong>
    Error message
    :Â 
  </strong>
  Failed to fetch
</p>
`;

exports[`Error Message component should not render label 1`] = `
<p
  style={
    Object {
      "color": "red",
    }
  }
>
  Failed to fetch
</p>
`;
